src/main/groovy/io/seqera/wave/service/blob/transfer/TransferQueue.groovy

```diff
@@ -45,4 +45,8 @@ class TransferQueue {
         transferQueue.poll(QUEUE_NAME, timeout)
     }
 
+    String pollAndGetValue(Duration timeout){
+        def transferEntry = poll(timeout)
+        return transferEntry ? transferEntry.substring(transferEntry.indexOf(QUEUE_NAME) + QUEUE_NAME.length() + 1) : null
```

Comment:
This is a bit messy. Think we need to unify `BlobCacheInfo.id` and `BlobCacheInfo.jobName`

Reply:
sorry, I don't understand
the poll is returning transfer-queue/v1=<id>
and we only need id here
https://github.com/seqeralabs/wave/blob/401638800bf968737968473d1298234a0a942593/src/main/groovy/io/seqera/wave/service/blob/transfer/TransferManager.groovy#L79

Reply:
Ohh, I was misunderstanding it. I didn't realised it because i've tested with the "local" queue. Then it should handle both cases and please add a unit test

Reply:
But if so, why this is OK ? 

https://github.com/seqeralabs/wave/blob/37174aeab5557686141fe9be8095d16be31fa49b/src/test/groovy/io/seqera/wave/service/data/queue/RedisMessageQueueTest.groovy#L66-L77

Reply:
Yes, I have noticed that. I am debugging this right now
<img width="479" alt="Screenshot 2024-08-12 at 14 52 01" src="https://github.com/user-attachments/assets/46aee864-f998-4479-97df-3c24036eb89c">

Reply:
I have updated the poll method to return just the value